pkg/event/event_unmarshal_test.go

```diff
@@ -2,18 +2,20 @@ package event_test
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 	"time"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/redhat-cne/sdk-go/pkg/event"
+	"github.com/redhat-cne/sdk-go/pkg/event/ptp"
 	"github.com/redhat-cne/sdk-go/pkg/types"
 )
 
 func TestUnMarshal(t *testing.T) {
 	now := types.Timestamp{Time: time.Now().UTC()}
 	resource := "/cluster/node/ptp"
-	_type := "ptp_status_type"
+	_type := ptp.STATUS
```

Comment:
remove ptp.STATUS constant and use only valid API defined types
example EventPtpStateChange

Reply:
done